geomesa-blobstore/geomesa-blobstore-api/src/main/java/org/locationtech/geomesa/blob/api/Blob.java

```diff
@@ -37,9 +36,10 @@ public String getLocalName() {
 
     @Override
     public String toString() {
-        return Objects.toStringHelper(this)
-                .add("id", id)
-                .add("localName", localName)
-                .toString();
+        return "Blob{" +
```

Comment:
Hmm... would it be better to use a StringBuilder?  Does it matter?

Reply:
possibly... i just used intellij generate->toString to make this :D

Reply:
Hehehe, yeah, let's leave it.